### ISSUE TITLE
Option to disallow raw HTML

### ISSUE THREAD

**Issue by mattbasta** [andrerpena/react-mde#74]
Currently, a user can type something like this:

```
<div onmouseenter="alert('poop')">foo</div>
```

Which will cause all manner of chaos. It doesn't seem that there's a way to disable arbitrary HTML. Can this be made an option?

**Comment by andrerpena** [comment]
Thanks for reporting. At first I thought it would be just a `Showdown` option but it is not. Then I looked into their demo and your scenario doesn't work there.. However, they are NOT encoding HTML. I'm not sure what they are doing different.

![image](https://user-images.githubusercontent.com/1581429/36125136-206d2870-1053-11e8-950d-c0281c878530.png)

**Comment by andrerpena** [comment]
@mattbasta This weekend I'll experiment with HTML encoding the text area content before passing it to the Markdown preview. If it works fine I'll push it.. I'll keep this thread updated

**Comment by mattbasta** [comment]
Thanks! No rush from my end.

**Comment by andrerpena** [comment]
Unfortunately, [there's no simple way to do it](https://github.com/showdownjs/showdown/wiki/Markdown's-XSS-Vulnerability-(and-how-to-mitigate-it)). I've added a prop called `processHtml` that allows you to process the HTML before it is displayed, which is the recommended way to do it according to Showdown.

However, I did not have time to investigate a "recommended" library for doing so yet. I'll create another issue to deal with it.
